04-trees/slides-forests1.Rnw

```diff
@@ -55,6 +55,34 @@ Train on $B$ \textbf{bootstrap} samples of data $\D$:
 \includegraphics[width=0.8\textwidth]{figure_man/rf_agg_majvot.pdf}
 \end{center}
 
+\framebreak
+
+\begin{footnotesize}
+\textbf{Example:}
+\\
+Let's say you have a binary classification task and your submodels are pretty bad ($acc = 0.55$).
+\\
+If all models would be very similar to each other, then they would all classify the \textbf{same} observations right or wrong.
+\\
+If all submodels are very different, but still have an accuracy-rate of $> 0.5$, then every new observation has a chance to be classified right by each sub-model with approximately this accuracy-rate.
+\\
+Now, let's say you have 1000 observations and 500 completely different sub-models with an accuracy-rate of $acc = 0.55$ each:
+
+<<echo = FALSE, eval = TRUE>>=
+
+set.seed(25092019)
+
+@
+
+<<echo = TRUE, eval = TRUE>>=
```

Comment:
no R code on the slides, please.

Reply:
I would change this into a two-colored histogram with the border at 250/251. do you use a specific ggplot theme for the slides?

Reply:
we do -- the setup is here: 
https://github.com/compstat-lmu/lecture_i2ml/blob/28393bb1b762f485688df6807b92ce0aeaea08c5/style/setup.Rnw#L78-L112
 this should work pretty automatically since this chunk is loaded before the slide is rendered.